File: src/pokerbots/spam_framework/PlayerHistory.java
package pokerbots.spam_framework;

import java.util.ArrayList;

public class PlayerHistory {

	private ArrayList<String> lastActions;
	
	public PlayerHistory() {
		
	}

	public ArrayList<String> getLastActions() {
		return lastActions;
	}

	public void setLastActions(ArrayList<String> lastActions) {
		this.lastActions = lastActions;
	}
}

File: spambot/SPAMFramework/PlayerHistory.cpp
#include "PlayerHistory.h"

PlayerHistory::PlayerHistory(){
}

std::vector<std::string> PlayerHistory::GetLastActions(){
	return m_last_actions;
}

void PlayerHistory::SetLastActions(std::vector<std::string> last_actions) {
	m_last_actions = last_actions;
}
